da/grpc/grpc.go

```diff
@@ -35,7 +37,8 @@ var DefaultConfig = Config{
 var _ da.DataAvailabilityLayerClient = &DataAvailabilityLayerClient{}
 var _ da.BlockRetriever = &DataAvailabilityLayerClient{}
 
-func (d *DataAvailabilityLayerClient) Init(config []byte, kvStore store.KVStore, logger log.Logger) error {
+func (d *DataAvailabilityLayerClient) Init(config []byte, _ store.KVStore, logger log.Logger) error {
```

Comment:
I definitely don't think that it should be done in this PR, but it might make sense to use a toml like the other configs instead of a json. I think it would also be cool to have a modified `init` function that would initialize optimint specific

Reply:
TBH, I was absolutely pragmatic while choosing JSON - I wanted to be able to pass config via commandline without newlines ;)

Reply:
makes sense! I think that was the best decision.